build.sbt

```diff
@@ -158,6 +159,19 @@ lazy val contribs = (project in file("contribs"))
     Compile / compile := ((Compile / compile) dependsOn createTargetClassesDir).value
   )
 
+// TODO javastyle tests
+// TODO unidoc
+lazy val storage = (project in file("storage"))
+  .settings (
+    name := "delta-storage",
+    commonSettings,
+    releaseSettings,
+    libraryDependencies ++= Seq(
+      // User can provide any version >= 3.1.0 (released in 2018). We don't use any new fancy APIs.
```

Comment:
We should be able to support any 2.x or 3.x version.

Reply:
do you want me to update the comment or update the actual dependency?

Reply:
just update the comment